src/components/MyLoader.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from "react";
+import { Container, Segment, Dimmer, Loader, Image } from "semantic-ui-react";
+import styled from "styled-components";
+
+const StyledContainer = styled(Container)`
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  margin: 1rem;
+`;
+
+const MyLoader = ({ isActive }) => (
```

Comment:
Don't use `My`, `Custom`, etc. prefixes.

Just call it `Loader` or `Spinner`

If you have a collision with import you can:
```
 import { Container, Segment, Dimmer, Loader as SemanticLoader, Image } from "semantic-ui-react";
```